src/vs/workbench/parts/quickopen/browser/gotoLineHandler.ts

```diff
@@ -91,12 +91,14 @@ class GotoLineEntry extends EditorQuickOpenEntry {
 
 		// Inform user about valid range if input is invalid
 		const maxLineNumber = this.getMaxLineNumber();
-		if (this.invalidRange(maxLineNumber)) {
+		const line = this.editorService.activeTextEditorWidget.getPosition().lineNumber;
```

Comment:
@cleidigh I suggest to move this into the body below, e.g.:

```ts
getLabel(): string {

	// Inform user about valid range if input is invalid
	const maxLineNumber = this.getMaxLineNumber();
	if (this.invalidRange(maxLineNumber)) {
		const currentLine = this.editorService.activeTextEditorWidget.getPosition().lineNumber;

		if (maxLineNumber > 0) {
			return nls.localize('gotoLineLabelEmptyWithLimit', "Current Line: {0}. Type a line number between 1 and {1} to navigate to.", currentLine, maxLineNumber);
		}

		return nls.localize('gotoLineLabelEmpty', "Current Line: {0}. Type a line number to navigate to.", currentLine);
	}

	// Input valid, indicate action
	return this.column ? nls.localize('gotoLineColumnLabel', "Go to line {0} and character {1}", this.line, this.column) : nls.localize('gotoLineLabel', "Go to line {0}", this.line);
}
```

Reply:
Education:
Is the advantage that the interpreter frees up the local block 'line' var at the close of the block?

Reply:
👍